airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -48,6 +48,7 @@
 from botocore.credentials import ReadOnlyCredentials
 from botocore.waiter import Waiter, WaiterModel
 from dateutil.tz import tzlocal
+from pytest import importorskip
```

Comment:
Pytest it is not a core dependency of Airflow, only development dependency.
And I’m not sure that is a good idea to use testing framework outside of the tests